test/constants/evaluation-tags-test.contant.ts

```diff
@@ -0,0 +1,54 @@
+import {CreateEvaluationTagDto} from '../../src/evaluation-tags/dto/create-evaluation-tag.dto';
+import {EvaluationTag} from '../../src/evaluation-tags/evaluation-tag.model';
+import {EvaluationTagDto} from '../../src/evaluation-tags/dto/evaluation-tag.dto';
+import {UpdateEvaluationTagDto} from '../../src/evaluation-tags/dto/update-evaluation-tag.dto';
+
+/* eslint-disable @typescript-eslint/ban-ts-ignore */
+
+// @ts-ignore
```

Comment:
Why does this need to be @ts-ignored? All I could find online was:

> A // @ts-ignore comment suppresses all errors that originate on the following line. It is recommended practice to have the remainder of the comment following @ts-ignore explain which error is being suppressed.

Reply:
It's missing properties from a full EvaluationTag object so this appeases the compiler